src/actions/notificationsActions.js

```diff
@@ -145,7 +145,7 @@ export const subscribeToPushNotificationsAction = () => {
       if (disabledFetchingBalanceListener !== null) return;
       disabledFetchingBalanceListener = setInterval(() => {
         dispatch(fetchAssetsBalanceNotificationsAction());
-      }, 10000);
+      }, 20000);
```

Comment:
`fetchAssetsBalanceNotificationsAction` - what does this actually do? Just curious to know why we're firing it every 20s...

Reply:
Actually, firing it to get token balance. Aldin required a quick refresh balance after a swap or send (within 10 seconds). And we have no token received notification state. So I set an interval of 10 seconds to get the updated balance. But I thought that every 10-second could affect another process.

Reply:
Before the set interval, it was fired every 30 seconds

Reply:
the interval is canceled only when the app state changes(Sing out or expire session).
Actually, we do have not any notification SDK method to find out whether received a token.

Reply:
We might need to try and figure out a mechanism to cancel this. I would recommend that we listen for the response, and as soon as we get it we attempt to cancel the interval. That is preferred, but the simple way is to cancel after a timeout

Reply:
@IAmKio I made changes as per your suggestion. please review it.
https://github.com/pillarwallet/pillarwallet/pull/3275

Reply:
We can resolve this issue due to the other PR